scripts/generate-clients.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
It would be better to create separate repo(s) for generated vmaas and inventory clients and use it as libs. It avoids to storing scripts like this one here.
Btw. it does not work for me:
~~~
[main] INFO  o.o.codegen.DefaultGenerator - Model Facts not generated since it's a free-form object
[main] INFO  o.o.codegen.AbstractGenerator - writing file /local/_generated/cmsfr/inventory/model_bulk_host_out.go
Exception in thread "main" java.lang.RuntimeException: Could not generate model 'BulkHostOut'
	at org.openapitools.codegen.DefaultGenerator.generateModels(DefaultGenerator.java:517)
	at org.openapitools.codegen.DefaultGenerator.generate(DefaultGenerator.java:923)
...
[main] INFO  o.o.codegen.AbstractGenerator - writing file /local/_generated/cmsfr/vmaas/model_cves_request.go
Exception in thread "main" java.lang.RuntimeException: Could not generate model 'CvesRequest'
~~~

Reply:
Moved the clients temporarily to https://github.com/semtexzv/patchman-clients, until we have repo in the RedhatInsights org. 

The error seems to be caused by user access rights to the directory. I'm running under docker, so its running with root .